lib/api/src/version.ts

```diff
@@ -1 +1 @@
-export const version = '5.2.0-alpha.39';
```

Comment:
@ndelangen Please don't modify this file. It only causes merge conflicts down the line.